config.js

```diff
@@ -374,6 +374,12 @@ var config = {
     // transcription: {
     //     // Whether the feature should be enabled or not.
     //     enabled: false,
+    
+    //     //Available languages for the language selector dialog.
+    //     translationLanguages: ["en", "es", "fr", "ro" ...]
```

Comment:
Make the comment actual working JS if uncommented please.

Reply:
done